### ISSUE TITLE
Plugin exports "postcssFontGrabber", not itself

### ISSUE THREAD

**Issue by CREEATION** [AaronJan/postcss-font-grabber#8]
I [copied this snippet](https://github.com/AaronJan/postcss-font-grabber/blob/master/README.md#with-gulp) from the readme, but the plugin doesn't export itself, see https://github.com/AaronJan/postcss-font-grabber/blob/master/src/index.ts#L35

This forces me to either write

```js
const postcssFontGrabber = require("postcss-font-grabber").postcssFontGrabber
```

or

```js
const { postcssFontGrabber } = require("postcss-font-grabber")
```

or, going way overboard with

```js
const {
  postcssFontGrabber: customFontGrabberName,
} = require("postcss-font-grabber")
```

If you can't get around to updating the readme I'd be open to issue a new pull request sometime this week.

Otherwise, if it's not actually a readme but a general code issue, I wouldn't consider me a fit for fixing this. :(

Thanks for the plugin 💚🐛 

-- Thomas

**Comment by AaronJan** [comment]
@CREEATION Thanks for your feedback!

When I migrating postcss-font-grabber to TypeScript, I switched to the default exports syntax, which broke that snippet.

It is indeed a documentation issue, PR welcomed!

**Comment by AaronJan** [comment]
Resolved in `v2.2.0` (now it supports CommonJS, AMD style: `const postcssFontGrabber = require('postcss-font-grabber')`), thanks for your contribution again! 🤝
